src/dev/ci_setup/setup_env.sh

```diff
@@ -119,6 +119,10 @@ yarn config set yarn-offline-mirror "$cacheDir/yarn-offline-cache"
 ###
 yarnGlobalDir="$(yarn global bin)"
 export PATH="$PATH:$yarnGlobalDir"
+# In some situations looks lik the yarn binaries added through
+# yarn global add will not be placed on yarn global bin
+# but on $HOME/.yarn/bin. I'm also adding it into the PATH
+export PATH="$PATH:$HOME/.yarn/bin"
```

Comment:
Is the issue that during the build bazelisk hasn't been installed? https://kibana-ci.elastic.co/job/elastic+kibana+pipeline-pull-request/104152/execution/node/453/log/

Reply:
It isn't I've tried that before at https://github.com/elastic/kibana/pull/90328/commits/ab7b5f32bd4f04ba1940ee867b7e401b3aef3df0 and the problem persisted even by installed bazelisk in place on those scripts. The problem is only related by not being exposing the correct folder to the PATH that stores the yarn binaries.

I'v also checked with Brian and by the time the workers runs the build Bootstrap already happened so Bazelisk was installed.

Reply:
I've applied the changes we discussed and moved the bazelisk installation process into npm